### ISSUE TITLE
How to set the height ratio of multiple plots in graphicslayoutwidget?

### ISSUE THREAD

**Issue by lyl1836** [pyqtgraph/pyqtgraph#1673]
I embed pyqtgraph in pyqt5. 

The structure of widget is as follows:

QVBoxLayout --> GraphicsLayoutWidget --> addPlot，

How do multiple addplots adjust the height scale?

I want the 4:1 display . The main codes are as follows:
`plot = self.plotview.addPlot(0,1,4,1, name='p1')
 plot1 = self.plotview.addPlot(row=5, col=0)`

But the actual effect is not as expected, as shown in the following figure:

**Comment by ixjlyons** [comment]
One way I can think of:

```python
import pyqtgraph as pg

app = pg.mkQApp()

glw = pg.GraphicsLayoutWidget()
glw.show()

glw.addPlot(x=[1, 2, 3], y=[1, 2, 3])
glw.nextRow()
glw.addPlot(x=[1, 2, 3], y=[1, 8, 27])

glw.ci.layout.setRowStretchFactor(0, 2)
# |  |  |      |
# |  |  |      `- https://doc.qt.io/qt-5/qgraphicsgridlayout.html#setRowStretchFactor
# |  |  `- QGraphicsGridLayout
# |  `- GraphicsLayout
# `- GraphicsLayoutWidget

app.exec_()
```

This has been asked on the [mailing list before](https://groups.google.com/g/pyqtgraph/c/aMaCMOrrrJg/m/Fcw2XPNJcvYJ) and seems like it'd be a somewhat common thing to want to do. One thing we could do here is add a `getLayout` method (to both `GraphicsLayout` and `GraphicsLayoutWidget`). For the latter, it'd be nice to link to https://doc.qt.io/qt-5/qgraphicsgridlayout.html

#1631 should also be helpful figuring things like this out

**Comment by lyl1836** [comment]
@ixjlyons :

Teacher, you are wonderful! Thank you very much.

But I am distressed: how can I find such information and solve such problems by myself!

For example: glw.ci This kind of grammar can't be found on the Internet!

**Comment by ixjlyons** [comment]
Right, my comments above about accessing the underlying layouts of `GraphicsLayout` and `GraphicsLayoutWidget` and documenting that access refer to things we could do to help. I'll keep this open as it indicates a gap in the API/docs.

**Comment by lyl1836** [comment]
@ixjlyons :

Test success, thank you very much!
